src/wrappers/React.js

```diff
@@ -24,12 +24,12 @@ const makeReactContainer = Component => {
     }
 
     render () {
-      const { children, ...rest } = this.state
+      const { children, '': invoker, ...rest } = this.state
       const wrappedChildren = this.wrapVueChildren(children)
 
       return (
         <Component {...rest}>
-          <VueWrapper component={wrappedChildren} />
+          { children ? <VueWrapper component={wrappedChildren} /> : null }
```

Comment:
1. It's probably better to write `children && <VueWrapper component={wrappedChildren} />`
2. It's not really mentioned anywhere, but code style this project uses treats JSX curly braces like template string interpolation, so there should not be leading and trailing whitespaces inside.

Reply:
🆒